Model/Page.php

```diff
@@ -174,6 +174,11 @@
     protected static $slugifyMethod;
 
     /**
+     * @var int
+     */
+    private $targetRedirectType = 302;
```

Comment:
That was my first thought too but I cleared all the caches and also deleted the cache folders.